src/client/java/teammates/client/scripts/DataMigrationForSanitizedDataHelper.java

```diff
@@ -0,0 +1,67 @@
+package teammates.client.scripts;
+
+public final class DataMigrationForSanitizedDataHelper {
+
+    private DataMigrationForSanitizedDataHelper() {
+        // utility class
+    }
+
+    /**
+     * Returns true if string contains no characters to be sanitized and contains substrings of sanitized characters
+     */
+    public static boolean isSanitizedHtml(String string) {
+        if (string == null) {
+            return false;
+        }
+        if (isTextContainingAny(string, "<", ">", "\"", "/", "\'")) {
+            return false;
+        }
+        return isTextContainingAny(string, "&lt;", "&gt;", "&quot;", "&#x2f;", "&#39;", "&amp;");
+    }
+
+    /**
+     * Checks if text contains any of the strings given.
+     * @return true if text contains at least one of the strings,
+     * false if text does not contain any of the strings or if no strings are given.
+     */
+    public static boolean isTextContainingAny(String text, String... strings) {
+        for (String string : strings) {
+            if (text.contains(string)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+
+    /**
+     * Utility class to be used with loops to print messages regularly.
+     */
+    public static class LoopHelper {
```

Comment:
This sounds like a utility class that can be reused by future scripts. Extract into a separate file?